src/matrix.rs

```diff
@@ -303,8 +303,9 @@ impl Matrix4 {
         )
     }
 
-    /// Create a 2d skew matrix
-    pub fn create_skew(sx: f32, sy: f32) -> Matrix4 {
+    /// https://drafts.csswg.org/css-transforms/#funcdef-skew
```

Comment:
I'd prefer to keep the descriptive doc string, and add the link to the body of the comment